generic_config_updater/patch_sorter.py

```diff
@@ -562,7 +562,8 @@ def __init__(self, config_wrapper):
 
     def validate(self, move, diff):
         simulated_config = move.apply(diff.current_config)
-        return self.config_wrapper.validate_config_db_config(simulated_config)
+        is_valid, error = self.config_wrapper.validate_config_db_config(simulated_config)
```

Comment:
>error [](http://example.com/codeflow?start=18&length=5)

Not a big issue, you can use `_` if you intend not to use it.

Reply:
Will fix it in a later PR as this is blocking https://github.com/Azure/sonic-utilities/pull/2145